src/Tasks/Microsoft.NET.Build.Tasks/SingleProjectInfo.cs

```diff
@@ -28,7 +40,11 @@ public IEnumerable<ResourceAssemblyInfo> ResourceAssemblies
             get { return _resourceAssemblies; }
         }
 
-        private SingleProjectInfo(string projectPath, string name, string version, string outputName, List<ReferenceInfo> dependencyReferences, List<ResourceAssemblyInfo> resourceAssemblies)
+        protected SingleProjectInfo()
+        {
+        }
```

Comment:
I think I might have needed to have the private constructor in the base class. But I might also have just forgotten about it :(

Reply:
Ah, I remembered. Initially I added it to have the private constructor, but then I realized I wanted the super class state to be initialized with empty collections (to avoid null refs), so I ended up using a different super constructor. But I forgot to delete the protected no-arg one.